app/components/e-progress.md

```diff
@@ -0,0 +1,29 @@
+### Styles
```

Comment:
e-progress is not rendering in styleguide.

Reply:
This is caused by an invalid vue-styleguidist version, which did not support mixins. Please reinstall node packages while in branch and check again.